code/game/objects/items/handcuffs.dm

```diff
@@ -368,7 +368,8 @@
 		C.update_inv_legcuffed()
 		SSblackbox.record_feedback("tally", "handcuffs", 1, type)
 		to_chat(C, "<span class='userdanger'>\The [src] ensnares you!</span>")
-		C.Paralyze(knockdown)
+		if(knockdown>0)
+			C.Paralyze(knockdown)	
```

Comment:
```suggestion
		if(knockdown)
			C.Paralyze(knockdown)	
```

Reply:
how can knockdown even be less than one 🤔

Reply:
slight oversight, the idea here is that since the knockdown causes paralyze bolas that have knockback set to 0 (the handmade) will not paralyze but instead, well, not paralyze.